### PR TITLE
Release 8.0.4

### PR DESCRIPTION
**Pull request by FabienPapet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x 
| Description?      | 
| Type?             |  improvement
| Category?         | CO
| BC breaks?        |  no
| Deprecations?     | no

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 class AppKernel extends Kernel
 {
-    const VERSION = '8.0.3';
+    const VERSION = '8.0.4';
```
Comment (matks) [inline app/AppKernel.php:35]: You need to also modify line 39 😉

**Review by matks — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by 0x346e3730 — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]
Thanks @FabienPapet

**Review by jolelievre — APPROVED** [review]
Thanks @FabienPapet

**Comment by jolelievre** [comment]
Running campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4690955009

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**